lib/index.js

```diff
@@ -3148,6 +3148,73 @@ Stream.prototype.reduce1 = function (f) {
 exposeMethod('reduce1');
 
 /**
+ * Reduces a stream, using a stream-returning function. This allows async
```

Comment:
Doesn't have to, multiple values just means the reducer will get called a bunch of times.